tools/autograd/derivatives.yaml

```diff
@@ -623,7 +623,7 @@
 
 - name: mv(Tensor self, Tensor vec) -> Tensor
   self: grad.ger(vec)
-  vec: self.t().mv(grad)
+  vec: "self.t().mv(grad.device().is_cpu() ? grad : grad.clone())"
```

Comment:
Note that the semantic of `.clone()` has changed, it now preserves the layout of the input.
If you just want a contiguous Tensor, you should use `grad.contiguous()` (that will avoid the copy in the case where it already is contiguous as extra benefit).

Reply:
Thanks for the information. Have removed `clone` in favour of `contiguous`.